lib/MoneyMarketAccount.rb

```diff
@@ -0,0 +1,93 @@
+module Bank
+  require "./account.rb"
+
+  class MoneyMarketAccount < Account
+
+    attr_accessor :num_transactions
+
+    def initialize(id, date)
+      super
+      @num_transactions = 0
+    end
+
+    def initial_balance
+      super(1000000)
+    end
+
+    def withdraw(amount)
+      if check_transactions && check_min_balance
```

Comment:
Where is the `check_transactions` variable set? It seems like it is used as a local variable here but I don't see where it would be assigned a variable prior to this line of code.